src/app/components/favorites/favorites.component.html

```diff
@@ -0,0 +1,22 @@
+<!-- favorites.component.html
+     displays list of favorite jokes -->
```

Comment:
Great use of simple, and descriptive comments. A better place for this though would be `favorites.component.ts` as a JSDoc comment for the class.